observer/storage/redis/storage.go

```diff
@@ -146,6 +146,7 @@ func (s *Storage) Delete(subs []observer.Subscription) error {
 
 func (s *Storage) updateWebHooks(subs []observer.Subscription, operation webHookOperation) error {
```

Comment:
when is this called?

Reply:
[here](https://github.com/trustwallet/blockatlas/pull/306/files#diff-1696d28c4b8857cb3cda676c006721b4R140) and [here](https://github.com/trustwallet/blockatlas/pull/306/files#diff-1696d28c4b8857cb3cda676c006721b4R144)